astro/src/content/docs/get-started/run-in-the-cloud/setup-wizard.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Setup Wizard & First Login
+title: Setup Wizard & First Time Setup
```

Comment:
It's been deemed that First-time Setup is the grammatically correct way of writing this. We could either leave this for now since it matches what's in the app and screen shots, or we could update it in the headings now.